site/ja/tutorials/load_data/tf-records.ipynb

```diff
@@ -0,0 +1,1687 @@
+{
```

Comment:
<p>The original document seems be changed:</p><ul><li>The "## TFRecords Format Details" section was added before this section.</li></ul>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/docs/pull/427/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>